Scripts/Runtime/Core/AnimationSequencerController.cs

```diff
@@ -22,11 +28,16 @@ public enum PlayType
         [SerializeField]
         private bool timeScaleIndependent = false;
         [SerializeField]
-        private bool playOnAwake;
+        private AutoplayMode autoplayMode = AutoplayMode.Awake;
+        public AutoplayMode AutoPlayMode { get { return autoplayMode;} set { autoplayMode = value; } }
```

Comment:
Can you create a `SetAutoPlayMode();` method instead of a setter?

Reply:
Hey Bruno, I will remove the setter for now without adding the SetAutoPlayMode() function as it is not actually used.

Reply:
Entire property has been removed.